cmd/loop/loopout.go

```diff
@@ -122,14 +136,41 @@ func loopOut(ctx *cli.Context) error {
 		return err
 	}
 
+	if ctx.IsSet("addr") && ctx.IsSet("account") {
+		return fmt.Errorf("cannot set --addr and --account at the " +
+			"same time. Please specify only one source for a new " +
+			"address to sweep the loop amount to")
+	}
+
 	var destAddr string
+	var account string
 	switch {
 	case ctx.IsSet("addr"):
 		destAddr = ctx.String("addr")
+
+	case ctx.IsSet("account"):
+		account = ctx.String("account")
+
 	case args.Present():
 		destAddr = args.First()
 	}
 
+	if ctx.IsSet("account") != ctx.IsSet("account_addr_type") {
```

Comment:
nit: similarly could be moved before the switch.

Reply:
+1